repo/packages/M/marathon/300/config.json

```diff
@@ -345,12 +345,12 @@
         },
         "metrics-datadog-transmission-interval-ms" : {
           "description" : "A transmission interval in milliseconds for the DataDog reporter.",
-          "type" : "long",
+          "type" : "integer",
           "default" : 10000
         },
         "metrics-histogram-reservoir-highest-trackable-value" : {
           "description" : "The highest trackable value of histograms and timers.",
-          "type" : "long",
+          "type" : "integer",
```

Comment:
was this tested? Since this undergoes multiple parsings by CLI, UI, Cosmos (and now package registry) it would be nice to make sure `3600000000000` works without loss of any precision with `integer`. If it doesn't work, we can use `number` too.